src/main/java/com/hm/petmaster/command/SetColorCommand.java

```diff
@@ -0,0 +1,81 @@
+package com.hm.petmaster.command;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+import org.bukkit.entity.Player;
+
+import com.hm.petmaster.PetMaster;
+import java.io.File;
+import java.io.IOException;
+import java.util.logging.Level;
+import org.bukkit.DyeColor;
+import org.bukkit.configuration.ConfigurationSection;
+import org.bukkit.configuration.InvalidConfigurationException;
+import org.bukkit.configuration.file.FileConfiguration;
+import org.bukkit.configuration.file.YamlConfiguration;
+
+/**
+ * Class in charge of handling player requests to change ownership of a pet
+ * (/petm setowner).
+ *
+ * @author Pyves
+ *
+ */
+public class SetColorCommand {
+
+	private static final String COLOR_CONFIG_NAME = "color";
+
+	private final PetMaster plugin;
+	private final File playerColorConfig;
+
+	public SetColorCommand(PetMaster plugin, File playerColorConfig) {
+		this.plugin = plugin;
+		this.playerColorConfig = playerColorConfig;
+	}
+
+	public void setColor(Player player, String[] args) {
+		if (args.length == 2) {
+			DyeColor color = DyeColor.valueOf(args[1]);
```

Comment:
This will throw `IllegalArgumentException` if the colour specified by the player is invalid, or not uppercase for example. I think it would be better user experience to loop through the enum values in a case insensitive manner, and return an error message with possible colour values if no matches are found. :wink: